packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -100,6 +101,7 @@ class GoogleMap extends StatefulWidget {
     this.zoomGesturesEnabled = true,
     this.liteModeEnabled = false,
     this.tiltGesturesEnabled = true,
+    this.tiltControlsEnabled = true,
```

Comment:
I'd suggest: `web45DegreeImageryEnabled`, and default to `false`. This has more to do with the 45 degree imagery feature (you need to be in satellite mode, fully zoomed in, in a supported city...) rather than the WebGL tilt feature (which looks more like the standard "tilt by dragging two fingers" in the mobile app)

Reply:
Will 45 degree imagery never exist on mobile? The gesture handling made sense to make very clear it's web only because that will never be on mobile, but I see no reason 45 degree imagery couldn't exist on mobile.

Reply:
@Rexios80 I don't know about Maps' plans, but I currently cannot enable 45 degree imagery in San Francisco in Google Maps itself (there's a WebGL "Globe Mode" and... a compass control on the web, haha). You need to Ctrl+Drag to change the perspective of the globe.

My feeling is that we could completely disable 45 degree imagery, and not many people would complain? (I proposed leaving the webOnlyFlag for it to be re-enabled, so we don't remove functionality from the web SDK for those who might want it)

Reply:
Also defaulting to false isn't a good idea right now because of the behavior I've noticed with this disabled. Per my own doc comment:

> Disabling this will cause map dragging to be interrupted if the [GoogleMap] is rebuilt while dragging.

My initial guess is setting the map tilt to 0 every time the map is drawn interrupts dragging. I can try to put something together to prevent this, but I'm not sure my solution will be good enough to merge in.

Reply:
@Rexios80 I've followed internally WRT 45 degree imagery, see what we can do about it.

About the dragging being interrupted by the map re-painting, we probably should take a look. Can you prepare a minimal repro example? I guess having a map with `tilt:0` in a scrollable list is enough for the issue to happen? (I also guess that's why you added the WebGestureHandling, to try and prevent that behavior?)

Reply:
Yeah honestly the more I play with it the more I'm not sure the plugin can deal with this itself. Setting `gmaps.MapOptions.rotateControl = false` disables the button for 45 degree imagery, but zooming in far enough still shows 45 degree imagery regardless. Setting `gmaps.MapOptions.tilt = 0` once isn't enough, because the tilt value is actually changing to 45 once zoomed in enough. And as already said, setting the tilt value interrupts map dragging, so the plugin can't just do this itself every map draw. We could maybe put something in the `onCameraMove` listener to check if the tilt value changed and force it back to 0, but that sounds vey hacky. Users of the plugin could do this themselves, but telling people to do that also seems very clunky since most everyone would want that behavior by default.

Reply:
@Rexios80 in my testing above, setting `tilt=0` and `rotateControl=false` was enough for the web sdk to not go into 45 degree mode. Then the only way to set the tilt to anything different than 0 was to call `setTilt(value)` as needed.

I've tested in this jsbin: https://jsfiddle.net/gh/get/library/pure/googlemaps/js-samples/tree/master/dist/samples/aerial-simple/jsfiddle which seems to have a functional API ready to test quickly :)

Reply:
Let me play with it more

Reply:
Alright I think I finally got it working

Reply:
What was holding me up is the map tilt value was reporting to be 0 by default, so I didn't think setting it to 0 when it already was would do anything 🙃

Reply:
Let me know what you think of the property name `fortyFiveDegreeImageryEnabled`. I really didn't want to tie it to web, because I don't see a reason 45 degree imagery couldn't exist on other platforms in the future and changing the name after merging this would be a breaking change.

Reply:
> What was holding me up is the map tilt value was reporting to be 0 by default, so I didn't think setting it to 0 when it already was would do anything 🙃

SAME, took me a few tries to add the value when the map was created.

> I don't see a reason 45 degree imagery couldn't exist on other platforms in the future

I've been told that that feature exists on "Raster" maps, but that mobile is using vector tiles (and web is going that way too). I like `fortyFiveDegreeImageryEnabled` BTW!